frontend/pulse3d/src/pages/uploads.js

```diff
@@ -14,30 +14,35 @@ let adminColumns = [
     name: "File Owner",
     center: true,
     sortable: true,
+    center: false,
```

Comment:
`center: true,` is two lines above this, couldn't you just change that line instead of adding all these new ones?